packages/react-components/theme-designer/src/ThemeDesigner.tsx

```diff
@@ -1,30 +1,40 @@
 import * as React from 'react';
 import type { ThemeDesignerProps } from './ThemeDesigner.types';
-import { useStyles } from './ThemeDesigner.styles';
-import { useThemeDesignerReducer } from './useThemeDesignerReducer';
+import { useStaticStyles, useStyles } from './ThemeDesigner.styles';
+import { AppState, DispatchTheme, initialAppState, useThemeDesignerReducer } from './useThemeDesignerReducer';
 import { teamsLightTheme, FluentProvider } from '@fluentui/react-components';
-
 import { Nav } from './components/Nav/Nav';
 import { Sidebar } from './components/Sidebar/Sidebar';
 import { Content } from './components/Content/Content';
 
+export type StateContext = {
+  appState: AppState;
+  dispatchAppState: React.Dispatch<DispatchTheme>;
+};
+
+export const AppStateContext = React.createContext<StateContext>({
```

Comment:
I think I'd recommend following the same naming convention we've been using in components, which has been something like this:

```
export type AppContextValue = {  etc }

export const AppContext = React.createContext<StateContextValue>( etc )
```